ui/src/main/java/edu/wpi/grip/ui/codegeneration/Exporter.java

```diff
@@ -149,14 +163,16 @@ private void generateCode(VelocityEngine ve, String templateDir, File file,
   /**
    * Code to generate the .h file if the export type is c++
    *
-   * @param ve The velocity engine used with the desired properties.
+   * @param ve          The velocity engine used with the desired properties.
    * @param templateDir The directory of the velocity templates that will be used.
-   * @param file The location to put the file.
-   * @param context The velocity context including the java files that will be used by the
-   *        templates.
+   * @param file        The location to put the file.
+   * @param context     The velocity context including the java files that will be used by the
+   *                    templates.
```

Comment:
Lot of whitespace changes, can you make sure checkstyle is being consistent and add a check if necessary to make sure that one of these is correctly enforced.